py/sync_examples.py

```diff
@@ -98,6 +194,30 @@ def make_gallery(examples: List[Example]):
 ''' + '\n' + '\n\n'.join([make_gallery_thumbnail(e) for e in examples])
 
 
+def make_tag_group(tag: str, examples: List[Example]) -> str:
+    example_links = [f"- <a href={{useBaseUrl('docs/examples/{e.slug}')}}>{e.title}</a>" for e in examples]
+    return f'### {tag}\n' + '\n' + '\n'.join(example_links) + '\n'
+
+
+def index_examples(examples: List[Example]):
+    tags_dict = defaultdict(list)
+    for e in examples:
+        for t in e.tags:
+            tags_dict[t].append(e)
+    return sorted(tags_dict.items())
```

Comment:
Need to sort by the example's title.

Reply:
@lo5 `label` is for `ui.label` and `labels` is for data labels in plots.

Reply:
Yes, in that case let's drop `labels`. Thanks @srini-x.